js/scripts.js

```diff
@@ -21,12 +21,43 @@ chevrons.forEach(function(chevron, index) {
 
 // newsroom tags
```

Comment:
We could add comments here.  
Will merge for now and try to refactor/add comments here if possible later.